CorsixTH/Lua/dialogs/fullscreen/staff_management.lua

```diff
@@ -67,9 +67,9 @@ function UIStaffManagement:UIStaffManagement(ui)
   -- Other buttons
   self:addPanel(0, 12, 86):makeButton(0, 0, 31, 74, 2, self.scrollUp):setTooltip(_S.tooltip.staff_list.prev_person)
   self:addPanel(0, 12, 274):makeButton(0, 0, 31, 74, 3, self.scrollDown):setTooltip(_S.tooltip.staff_list.next_person)
-  self:addPanel(0, 319, 372):makeButton(0, 0, 112, 39, 7, self.payBonus):setTooltip(_S.tooltip.staff_list.bonus)
-  self:addPanel(0, 319, 418):makeButton(0, 0, 112, 39, 8, self.increaseSalary):setTooltip(_S.tooltip.staff_list.pay_rise)
-  self:addPanel(0, 438, 372):makeButton(0, 0, 45, 85, 9, self.fire):setTooltip(_S.tooltip.staff_list.sack)
+  self:addPanel(0, 319, 372):makeButton(0, 0, 112, 39, 7, self.payBonus):setTooltip(_S.tooltip.staff_list.bonus):setSound() -- custom sound
```

Comment:
Can you explain these sound comments a little more, they don't make sense to me and they aren't describing the whole line.

Reply:
I see it. ... lets move and clarify the comments a little.

```
-- Disable the default sounds on the the following three buttons as their sounds are implemented in the callback.